docs/contributing/testing.md

```diff
@@ -0,0 +1,112 @@
+# Test your GOV.UK Frontend contribution
+
+Before you submit your contribution, you should:
+
+- check your changes look right and behave correctly
+- test in supported browsers and assistive technology
+- run our automated tests, and update tests or add new tests if our tests fail
+
+[Contact us](https://design-system.service.gov.uk/#support) for help if either:
+
+- you can’t do some of the checks or tests
+- your contribution might affect other parts of GOV.UK Frontend
+
+## 1. Lint your Sass and JavaScript
+
+Do the following to lint your Sass code.
+
+1. Install a linter in your code editor, for example [linter-sass-lint for Atom](https://atom.io/packages/linter-sass-lint).
+2. Point the linter to [GOV.UK Frontend’s `.sass-lint.yml` file](https://github.com/alphagov/govuk-frontend/blob/master/config/.sass-lint.yml).
+
+To lint your JavaScript code, install [Javascript Standard Style](https://standardjs.com/) and either:
```

Comment:
Hanna and I had a chat about all this, and we've ended up simplifying this down to running `npm test` as step 1, so that you see and fix the linting issues there. Hope that makes sense.